config/config.go

```diff
@@ -183,6 +187,54 @@ func (c *Config) SetLogLevel(logLevel string) {
 	c.logLevel = logLevel
 	logger.SetLogLevel(c.logLevel)
 }
+func (c *Config) SetLogFilePath(logPath string) {
+	file, err := getFile(logPath)
+	if err != nil {
+		logger.LogErrorWithLevel(messages.MsgErrorSettingLogFile, err)
+		return
+	}
+	logger.LogInfo("Set log file to " + file.Name())
+	c.logPath = filepath.Dir(file.Name())
+	logger.LogSetOutput(file, os.Stdout)
+}
+
+func getFile(logPath string) (*os.File, error) {
+	var err error
+	if logPath == "" {
+		logPath, err = getDirName()
+		if err != nil {
+			return nil, err
+		}
+	} else if _, err = os.Stat(logPath); os.IsNotExist(err) {
+		return nil, err
+	}
+	return createLogFile(logPath)
+}
+
+func createLogFile(dir string) (*os.File, error) {
+	fileName := dir + "/horusec-log-" + time.Now().Format("2006-01-02 15:04:05") + ".log"
```

Comment:
We should use `filepath.Join(dir, "horusec-log", ...)` instead. Concatenating paths with strings will broke on Windows.